yt_dlp/dependencies/__init__.py

```diff
@@ -43,6 +43,11 @@
 
 try:
     import sqlite3
+    # `sqlite3.version` is deprecated in py3.12, see https://github.com/yt-dlp/yt-dlp/issues/8152
+    import warnings as _warnings
+    with _warnings.catch_warnings():
+        _warnings.simplefilter('ignore', DeprecationWarning)
+        sqlite3._yt_dlp__version = getattr(sqlite3, 'version', '2.6.0+')
```

Comment:
we could display the runtime library version as well
```suggestion
    import warnings as _warnings
    with _warnings.catch_warnings():
        _warnings.simplefilter('ignore', DeprecationWarning)
        _sqlite_module_version = getattr(sqlite3, 'version', '2.6.0+')
    sqlite3._yt_dlp__version = f'{_sqlite_module_version} (sqlite {sqlite3.sqlite_version})'
```
&nbsp;
or since the python module version is kind of pointless, we could just do this

```suggestion
    sqlite3._yt_dlp__version = sqlite3.sqlite_version
```

Reply:
I like the second version, since it simplifies the handling by a lot